### DISCUSSION TITLE
React18 new API

### DISCUSSION THREAD

**Discussion by activeguild** [reactjs/react-rails#1247] [Q&A] [answered]
Are there any plans to incorporate the new react-dom/client API in react18?

**Comment by ahangarha** [comment, marked as answer]
Yes.
It is in planned to provide this support in the coming release.
I hope you have no issue more than the warning in the console. If so, please report it.

**Comment by activeguild** [comment]
@ahangarha 
That's amazing!
I am looking forward to it.
Thanks for all your efforts.

**Comment by thiagosf** [comment]
Awesome! 🤩

**Comment by lnazarit** [comment]
when is the next release ?
